src/components/emby-slider/emby-slider.js

```diff
@@ -177,6 +185,116 @@ define(['browser', 'dom', 'layoutManager', 'css!./emby-slider', 'registerElement
         }
     };
 
+    /**
+     * Keyboard dragging timeout.
+     * After this delay "change" event will be fired.
+     */
+    var KeyboardDraggingTimeout = 1000;
+
+    /**
+     * Keyboard dragging timer.
+     */
+    var keyboardDraggingTimer;
+
+    /**
+     * Start keyboard dragging.
+     *
+     * @param {Object} elem slider itself
+     */
+    function startKeyboardDragging(elem) {
+
```

Comment:
```suggestion
```

Reply:
Looking at the rest of the code, most of the blocks (functions) are separated from their beginning. Trying to understand the code style. So, there is no need to separate the beginning of the block from its body, isn't it?

Reply:
that's how I like to write stuff... but I mostly review JF web code for now, let's ask @jellyfin/web

Reply:
That's an inherited style :p most of us hate it but we just barely deminified most of the code, stylistic changes will be next. Generally, newlines should probably follow the rules below.

* no empty lines at the beginning of methods
* always have an empty line after code blocks (methods, if-else, switch, etc)
* never more than one empty line
* single newlines to break up logical sections of code as desired